frontend/src/views/Home.vue

```diff
@@ -260,26 +286,48 @@ export default {
   left: 0px;
 }
 
-#sponsor {
-  align-items: right;
-  padding-bottom: 150px;
+// SUPPORT CSESOC SECTION
+.sponsor {
+  margin-bottom: $space-xl;
+  background-image: url(../assets/supportusbackground.png);
+  background-size: cover;
+  background-position: 20% 50%;
 }
 
-#sponsor p {
-  font-size: 30px;
-  padding: 0 0 25px 0;
+.sponsor--h1 {
+  @extend h1;
```

Comment:
you don't need to extend for these because they are styles associated with the tag itself. So `<h1>` tag already comes with h1 styles.

Reply:
So if you could remove all occurrences of this type of extend.